docs/egress-service.md

```diff
@@ -324,6 +324,9 @@ This makes the egress traffic of endpoints of an EgressService to be routed via
 An ip rule is also created for the ClusterIP of the service which is needed in order for the return traffic (reply to an external client calling the service) to use the correct table - this is because the packet flow of contacting a LoadBalancer service goes:
 `lb ip -> node -> enter ovn with ClusterIP -> exit ovn with ClusterIP -> exit node with lb ip`
 
+### Dealing with non SNATed traffic
+TBD: currently a known issue, we can't leverage the same as [EgressIP](./egress-ip.md#Dealing-with-non-SNATed-traffic) currently does, the flow won't be hit because the traffic "exits" OVN here as opposed to how EgressIP "keeps everything" inside OVN.
```

Comment:
This needs more description. Describe the actual problem, then explain how it relates to egress IP.

Reply:
done